pandas/core/config_init.py

```diff
@@ -478,14 +479,19 @@ def use_inf_as_na_cb(key):
     _use_inf_as_na(key)
 
 
+# Get the default from an environment variable, if set, otherwise defaults to "block"
```

Comment:
i would just inline this on L494 (and add a 1-liner comment before) and/or update the option description with the text on how to control this